activerecord/lib/active_record/connection_adapters/postgresql/schema_definitions.rb

```diff
@@ -173,11 +173,19 @@ def primary_key(name, type = :primary_key, **options)
         # :method: xml
         # :call-seq: xml(*names, **options)
 
+        ##
+        # :method: timestamptz
```

Comment:
not related to this PR - forgot to add this line in https://github.com/rails/rails/pull/41395